base/datatypes.py

```diff
@@ -174,9 +173,10 @@ class Sweepable(XenonObj):
   # Per-instance lists can be modified, but this list cannot.
   sweepable_params = []
 
-  builtins_ = [IntType, FloatType, StringType, DictType, ListType, NoneType,
-               BooleanType, LongType, ComplexType, TupleType, UnicodeType,
-               UnassignedParamValue]
+  builtins_ = [
```

Comment:
These classes were added in https://github.com/xyzsam/xenon/commit/1454448f337141b7f8c0888113b5277c593e29df for validation purposes, we should continue to use them.

Reply:
Right. In Python3, these types are simplied to just int, float, etc. In particular, IntType and LongType now become int, StringType and Unicode become str, that's why those two are missing.

Reply:
> These classes were added in [1454448](https://github.com/xyzsam/xenon/commit/1454448f337141b7f8c0888113b5277c593e29df) for validation purposes, we should continue to use them.

Right. In Python3, these types are simplied to just int, float, etc. In particular, IntType and LongType now become int, StringType and Unicode become str, that's why those two are missing.